Dockerfile

```diff
@@ -105,6 +105,8 @@ RUN CHART_VERSION="2.11.100-build2021022302"  CHART_FILE=/charts/rke2-metrics-se
 RUN CHART_VERSION="v3.7.1-build2021041604"    CHART_FILE=/charts/rke2-multus.yaml         CHART_BOOTSTRAP=true   /charts/build-chart.sh
 RUN CHART_VERSION="1.0.000"                   CHART_FILE=/charts/rancher-vsphere-cpi.yaml CHART_BOOTSTRAP=true   CHART_REPO="https://charts.rancher.io" /charts/build-chart.sh
 RUN CHART_VERSION="2.1.000"                   CHART_FILE=/charts/rancher-vsphere-csi.yaml CHART_BOOTSTRAP=true   CHART_REPO="https://charts.rancher.io" /charts/build-chart.sh
+RUN CHART_VERSION="0.1.1"                     CHART_FILE=/charts/harvester-cloud-provider.yaml CHART_BOOTSTRAP=true   CHART_REPO="https://charts.rancher.io" /charts/build-chart.sh
```

Comment:
Do the charts have to be in the rancher repo? Are they used outside of the context of rke2?

We discovered with the vsphere chart that there is a bit of a disconnect between how rancher releases charts and how rke2 consumes them. 

The workflow for updating charts might be better if these charts lived in the rke2-charts repo

Reply:
currently, the PR is submitted against the rancher/charts repo https://github.com/rancher/charts/pull/1426, if rke2-charts is making much more sense we should be able to change that, cc @yaocw2020